packages/plugin-hardhat/src/admin.ts

```diff
@@ -36,8 +36,12 @@ export function makeChangeProxyAdmin(hre: HardhatRuntimeEnvironment, platformMod
     const proxyAdminAddress = await getAdminAddress(hre.network.provider, proxyAddress);
 
     if (manifestAdminAddress !== proxyAdminAddress) {
-      throw new Error('Proxy admin is not the one registered in the network manifest');
-    } else if (manifestAdminAddress !== newAdmin) {
+      console.log(
+        `Proxy admin ${proxyAdminAddress} is not the one registered in the network manifest ${manifestAdminAddress}`,
+      );
+    }
```

Comment:
This message can just be deleted, since the manifest's admin is not really relevant anymore for this scenario.